### PR TITLE
LPD-35329 add delivery group to address checkout step

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/frontend/data/set/provider/CommerceCheckoutDeliveryGroupFDSDataProvider.java [hunk 1]
```diff
@@ -38,19 +38,19 @@
  * @author Luca Pellizzon
  */
 @Component(
-	property = "fds.data.provider.key=" + CommerceCheckoutFDSNames.DELIVERY_GROUP,
+	property = "fds.data.provider.key=" + CommerceCheckoutFDSNames.DELIVERY_GROUPS,
```
Comment (smottal) [inline modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/frontend/data/set/provider/CommerceCheckoutDeliveryGroupFDSDataProvider.java:41]: if now this is plural is it correct that the class is singular CommerceCheckout**DeliveryGroup**FDSDataProvider? Also other classes has the name singular

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:forward:force

**Pull request closed without merging**